README.md

```diff
@@ -55,9 +65,23 @@ POLLY_MERGE_LOG_FILE=<your log file location>
 Work in progress, remaining stuff:
 
 - maybe rework configuration to be a little less lame (list of params?)
-- add some basic ci checks like black/pylint/sort imports at least
+- add fake bitbucket server api 1.0
 
 This should probably not be python but I'm reallllly lazy.
 
 Keeping it dependency free at least so it has a chance of running on someone's
 system.
+
+## Developing
+
+You can run whatever tests CI runs with these commands (requires Docker):
+
+```bash
```

Comment:
Maybe we should add a dedicated script for this (per usual)... that might help reduce the copy paste here and in the Github CI. It would also make it easier to test locally :thinking: ...

Reply:
SGTM, making that change.